services/QuillLMS/app/models/concerns/translatable.rb

```diff
@@ -77,11 +77,13 @@ def create_translation_mappings_with_text(translatable_text:, field_name: defaul
   end
 
   def translation(locale: DEFAULT_LOCALE, source_api: OPEN_AI_SOURCE)
-    @translation ||= translations(locale: locale, source_api: source_api)&.first&.translation
+    @translation ||= translations(locale: locale, source_api: source_api, field_name: default_translatable_field)&.first&.translation
```

Comment:
It feels a little redundant to have the default set in two places here.  I'd probably just have it in the method definition).